client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -205,7 +226,26 @@ const PatternAssembler: Step = ( { navigation } ) => {
 				<PatternSelectorLoader
 					showPatternSelectorType={ showPatternSelectorType }
 					onSelect={ onSelect }
-					onBack={ () => setShowPatternSelectorType( null ) }
+					onDoneClick={ () => {
+						const uniquePatterns = uniqueBy(
```

Comment:
Wondering why do we want unique patterns? Is there a requirement somewhere?

Reply:
hmm, no there is no requirement, I use my best guess, I just realized other list of names and ids did not need unique values...

https://github.com/Automattic/wp-calypso/blob/e2b81889276096b5fe01d6d982ea09216947d0ae/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx#L74

Reply:
Yeah, I also noticed that, what if we also don't unique the ids here as well for consistency?

Reply:
Removed unique filter 

![Screenshot on 2022-12-23 at 16-35-26.png](https://cdn-std.droplr.net/files/acc_1243635/orzhCT)